examples/java/spring-boot/src/main/java/uk/gov/api/springboot/domain/model/RegistryEntry.java

```diff
@@ -0,0 +1,3 @@
+package uk.gov.api.springboot.domain.model;
+
+public record RegistryEntry(String id, String baseUrl) {}
```

Comment:
I'd see this instead as `Entry`, inside the containing interface `Registry`, so we use it as `Registry.Entry`, then it's very clear it's only to be used within the context of the `Registry` - thoughts?

Reply:
yeah makes sense